### PR TITLE
Atom Dark Theme

### PR DESCRIPTION
**Pull request by gibsjose**
Added an Atom Dark Theme based on https://github.com/atom/atom-dark-syntax.

### CODE REVIEW

File: components.js [hunk 1]
```diff
@@ -30,6 +30,10 @@ var components = {
 			"title": "Coy",
 			"owner": "tshedor"
 		}
```
Comment (nauzilus) [inline components.js]: Missing a comma here

Comment (gibsjose) [inline components.js]: Thanks mate :)

**Comment by Golmote** [comment]
Could you submit this PR in the [Prism themes](https://github.com/PrismJS/prism-themes) repo please?

**Comment by gibsjose** [comment]
Sure thing!

**Pull request closed without merging**